veg2hab/vegkartering.py

```diff
@@ -789,12 +843,13 @@ def from_shapefile(
         shape_path: Path,
         *,
```

Comment:
Dit (en from_access, maar vooral deze) zijn echt draken van methods geworden met vanalles door elkaar, willen we dat nu of nog wat verder richting t einde nog eens even onder handen nemen? Aan de ene kant is t zo niet netjes, aan de andere kant maakt dat niet echt heel veel uit...

Reply:
Ja hier even een keer goed naar kijken, lijkt me wel een goede.